chapter_introduction/intro.md

```diff
@@ -1,1079 +1,2137 @@
 # Introduction
 
 Until recently, nearly all of the computer programs
-thay we interacted with every day were coded
-by software developers from first principles.
-Say that we wanted to write an application to manage an e-commerce platform.
-After huddling around a whiteboard for a few hours to ponder the problem,
+thay we
+interacted with every day were coded
+by software developers from first
+principles.
+Say that we wanted to write an application to manage an e-commerce
+platform.
+After huddling around a whiteboard for a few hours to ponder the
+problem,
```

Comment:
It seems that many sentences are broken into multiple lines. Can you keep the original sentences without breaking them?

Reply:
No. This is a better way to write.

Reply:
Wait something looks weird here---somehow things got garbled...

Reply:
Why better? It makes reviewing a bit hard. Most "changes" are identical so the real changes are hard to recognize.

Reply:
If you did 'pip install notedown' before, you can try 'pip uninstall notedown' then 'pip install https://github.com/mli/notedown/tarball/master'

The latest notedown may cause that problem if you use Jupyter Notebook to edit MD.